core/src/main/java/org/apache/carbondata/core/scan/result/vector/CarbonColumnVector.java

```diff
@@ -114,4 +114,14 @@
 
   void setLazyPage(LazyPageLoader lazyPage);
 
+  CarbonColumnVector getColumnVector();
```

Comment:
to get the original vector from wrapper ?

Reply:
done

Reply:
yes